ansible_collections/arista/avd/plugins/plugin_utils/cv_client/api/arista/subscriptions/__init__.py

```diff
@@ -8,7 +8,15 @@
 
 from dataclasses import dataclass
 
-import aristaproto
+try:
+    import aristaproto
+except ImportError:
+    HAS_ARISTAPROTO = False
+    from ....mocked_classes import mocked_aristaproto as aristaproto
+    from ....mocked_classes import mocked_grpclib as grpclib
```

Comment:
why is it importing the mocked grpclib here when it is not needed in the normal imports?

Reply:
Too much copy/paste. I will fix.

Reply:
fixed